releases/release-announcement-guide.md

```diff
@@ -0,0 +1,306 @@
+<p>
+Working Groups updates - 1 or 2 slides per WG
+<p>
+
+    Provide a format for slides - i.e. Info on WG Meeting (Time, Notes, Link) along with Top 3 feature names, short description, benefit and PR 
+<p>
+Distributions
+<p>
+Tutorials
+<p>
+What’s next
+<p>
+Getting Involved
+   </td>
+  </tr>
+  <tr>
+   <td>14 days
+<p>
+Write 2nd version of blog
+<p>
+Share 2nd version with the Working Group Leads and Release team in google doc.
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>12 days out
+<p>
+Schedule date for presentation,
+<p>
+Identify speakers for recorded presentation
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>11 days out
+<p>
+Review Working Group edits
+<p>
+Create and distribute 2rd version to Working Groups
+<p>
+Ask for final comments
+   </td>
+  </tr>
+  <tr>
+   <td>9 days out
+<p>
+Create 3rd version in google doc
+<p>
+Incorporate comments
+<p>
+Confirm the Links in doc
+<p>
+Note - Inputting links in a PR does take time and the number of links should be kept to a reasonable level.  Please confirm that the links are correct before creating the PR.
+<p>
+Move content from google doc to PR
+<p>
+Create PR using fast pages,https://github.com/kubeflow/blog#blog-submission-instructions
+<p>
+Ask for comments, edits from Working Groups leads on PR
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>7 days out
+<p>
+Update PR based on comments
+<p>
+Ask for final comments / final reviews
+<p>
+Confirm the Links
+<p>
+Ask Working Group leads for /lgtm
+<p>
+Alert Steering Committee that near final version is close
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>6 days out
+<p>
+Record presentation
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>5 days out
+<p>
+Review presentation, ask for any comments/edits, re-recordings
+<p>
+Provide to Video Pro and complete splice in edits.
+<p>
+<strong>Tools</strong>
+<p>
+Quicktime
+<p>
+Premiere pro
+<p>
+Handbrake
+<p>
+Sample Time - 1-3 days
+   </td>
+  </tr>
+  <tr>
+   <td>4 days out
+<p>
+Add in anything learned from presentation
+   </td>
+   <td>4 days out
+<p>
+Complete final edits and review
+<p>
+Provide /lgtm from content providers and approvers 
+   </td>
+  </tr>
+  <tr>
+   <td>3 days out
+<p>
+Get final /ltgm from stakeholders
+<p>
+Steering Committee adds Final edits
+   </td>
+   <td>3 days out 
+<p>
+Submit to Steering committee for posting on Kubeflow channel on YouTube
+   </td>
+  </tr>
+  <tr>
+   <td>Day of Post
+<p>
+Steering Committee approves and posts
+<p>
+Ask all Resources to review for any corrections
+<p>
+Have team ready for an quick updates
+<p>
+Promote on Kubeflow slack and on Kubeflow-discuss
+   </td>
+   <td>Day of Post
+<p>
+Highlight in presentation on Kubeflow-discuss mailing list (<a href="mailto:kubeflow-discuss@googlegroups.com">kubeflow-discuss@googlegroups.com</a> and on Kubeflow-slack (kubeflow.slack.com) in general channel
```

Comment:
- Could we also highlight the release in [kubeflow twitter](https://twitter.com/kubeflow)?
- Are we able to leverage the slack `#announcement` channel?